internal/pkg/api/handleCheckin.go

```diff
@@ -99,6 +108,7 @@ func NewCheckinT(
 		gcp:    gcp,
 		ad:     ad,
 		tr:     tr,
+		limit:  rate.NewLimiter(rt, cfg.Limits.ActionLimit.Burst),
```

Comment:
There is no `ActionLimit.Max` setting, does it mean only the `Burst` is being limited? What does an `Interval: 0.25ms` mean?

Reply:
interval and burst are used to configure the rate limiter (interval is the time it takes for 1 token to be added to the rate limit pool, burst is max pool size)
the max attributes was used to add limits to the total number of connections allowed on an endpoint (here we would use the checkin endpoint setting).